### PR TITLE
feat: updated status badge appearance and organisation

### PR DESCRIPTION
**Pull request by ckbedwell**
Because we are on a tight schedule I thought I'd make some of the change suggestions I had for the original branch (if we didn't have a hard deadline I'd have been more polite 😅)

I'll add the explanations for the changes in a self-review of the PR.

### CODE REVIEW

File: src/components/CheckForm/CheckForm.tsx [hunk 1]
```diff
@@ -166,22 +168,11 @@ export const CheckForm = ({ check, disabled, pageTitle }: CheckFormProps) => {
...
-                    </TextLink>
-                  </div>
-                </Alert>
-              )}
-
-              <FormLayout.Section label={checkTypeStep1Label[checkType]} fields={[`job`, ...defineCheckFields]}>
+              <FormLayout.Section
+                label={checkTypeStep1Label[checkType]}
+                fields={[`job`, ...defineCheckFields]}
+                status={status}
```
Comment (ckbedwell) [inline src/components/CheckForm/CheckForm.tsx:174]: I found the alert banner quite overwhelming and distracting from making a browser check (especially as there was no way to dismiss it and it was on every step of the form).

A screenshot of the add browser check form. The section title has a Private preview badge to the right of it.

File: src/components/CheckUsage.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
 });
 
-const hideTelemetryForTypes = [CheckType.Scripted, CheckType.MULTI_HTTP];
+const hideTelemetryForTypes = [CheckType.Scripted, CheckType.MULTI_HTTP, CheckType.Browser];
```
Comment (ckbedwell) [inline src/components/CheckUsage.tsx:34]: I thought I'd add this whilst I was there (I was considering adding the alert about them being free to this component but decided against it in the end).

File: src/hooks/useCheckTypeGroupOptions.tsx [hunk 3]
```diff
@@ -80,7 +79,6 @@ export const CHECK_TYPE_GROUP_OPTIONS: CheckTypeGroupOption[] = [
     description: `Monitor the availability and performance of a website using a real browser.`,
     value: CheckTypeGroup.Browser,
     icon: `globe`,
-    status: CheckStatus.PRIVATE_PREVIEW,
```
Comment (ckbedwell) [inline src/hooks/useCheckTypeGroupOptions.tsx:83]: Similar problem to above where I wanted all of the associated information colocated so it is easier to reason with.

File: src/components/ChooseCheckGroup.tsx [hunk 4]
```diff
@@ -40,6 +41,10 @@ export const ChooseCheckGroup = () => {
 const CheckGroupCard = ({ group }: { group: CheckTypeGroupOption }) => {
   const styles = useStyles2(getStyles);
   const { isOverCheckLimit, isOverScriptedLimit } = useLimits();
+  const checkOptions = useCheckTypeOptions().filter((option) => option.group === group.value);
+  const checksWithStatus = checkOptions.filter((option) => option.status);
+  const shouldShowStatus = checksWithStatus.length === checkOptions.length;
```
Comment (ckbedwell) [inline src/components/ChooseCheckGroup.tsx:46]: This actually resulted in the Public Preview badge appearing for Scripted checks which I thought was a good thing!

A screenshot of the Scripted and Browser check cards next to each other. The scripted card has a green Public preview badge beneath it. The Browser card has a purple Private preview badge beneath it.

**Review by ckbedwell — COMMENTED** [review]
Self-review.

File: src/components/CheckForm/CheckForm.tsx (hunk 1 above)
Comment (VikaCep) [reply src/components/CheckForm/CheckForm.tsx:174]: Love the improvement!

**Review by VikaCep — APPROVED** [review]
👏 👏 👏

**Pull request merged**